radixdlt/src/main/java/com/radixdlt/consensus/validators/ValidatorSet.java

```diff
@@ -36,11 +36,23 @@
  * as long as all validators sign.
  */
 public final class ValidatorSet {
+	// We assume that we won't have more than 2^128 validators (2^32 is in fact the limit)
+	// in a single validator set so having this as the max power value will prevent overflows
+	private static final UInt256 POWER_MAX_VALUE = UInt256.from(UInt128.MAX_VALUE);
 	private final ImmutableBiMap<ECPublicKey, Validator> validators;
+	private final transient UInt256 totalPower;
```

Comment:
I don't think we need to worry about "transient" modifier unless there is some likelihood that the object will be serialized in some way.

Reply:
Probably should be UInt384.

Reply:
`transient` is also used by EqualsVerifier to see what should actually be checked for equality

Reply:
Would probably prefer that we placate EqualsVerifier in the setup for EqualsVerifier, rather than in the code proper.

Reply:
Not sure I agree here. I think `transient` is clear indicator in code that parameter does not participate in equals and is most likely derived from other parts of the classes state. Also, a clear indicator if this class is to be serialized by some serialization mechanism in the future.

Reply:
No probs.  I can live with this.